hooks/pre-commit

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
 #
-# An example hook script to verify what is about to be committed.
-# The hook should exit with non-zero status after issuing an appropriate message if it wants to prevent the commit.
-#
-# In the project directory, run `git config core.hooksPath hooks/` to install the hooks of this 
-# directory (without affecting how git hooks work for other projects).
+# In the project directory, run `git config core.hooksPath hooks/` to install the hooks of this directory (without affecting how git hooks work for other projects).
+
+# To avoid having this pre-commit hook block your commits, configure your VSCode like this:
+
+# {
+#   "editor.formatOnPaste": true,
+#   "editor.formatOnSave": true,
+#   "editor.formatOnType": true,
+#   "editor.codeActionsOnSave": {
+#     "source.fixAll.eslint": true
+#   },
+#   "[rust]": {
+#     "editor.formatOnSave": true,
+#     "editor.defaultFormatter": "rust-lang.rust-analyzer"
+#   },
+#   //...
+# }
```

Comment:
Could go in `/.vscode/` or README; this is not a place most people would look for development environment configuration instructions.